### PR TITLE
feat: Use `poetry.core` to manage dependencies

### PR DESCRIPTION
**Pull request by tachikoma-li**
Changes proposed in this pull request:

- Remove dephell and setup.py
- Update README

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -75,6 +65,17 @@ Once the clone is complete, you have two options:
    poetry install
    ```
 
+1. Using pip
```
Comment (perjlieb) [inline .github/README.md:68]:
```suggestion
2. Using pip
```

Comment (tachikoma-li) [reply .github/README.md:68]: These are automatically numbered.

File: .github/README.md [hunk 2]
```diff
@@ -75,6 +65,17 @@ Once the clone is complete, you have two options:
...
+
+   ```shell
+   cd python-open-controls
+   python setup.py develop
```
Comment (leoadec) [inline .github/README.md:72]: Sorry Li, how does this line work if we're removing `setup.py`?

**Review by leoadec — COMMENTED** [review]
Looks good, just have a question

File: .github/README.md (hunk 2 above)
Comment (perjlieb) [inline .github/README.md:72]: That seems to be difficult to run without setup.py

**Review by leoadec — APPROVED** [review]
🎏

**Pull request merged**